Sources/Plasma/PubUtilLib/plFile/plSecureStream.cpp

```diff
@@ -325,9 +325,10 @@ uint32_t plSecureStream::IRead(uint32_t bytes, void* buffer)
         }
         else
         {
-            // TODO: errno equivalent for *nix
 #if HS_BUILD_FOR_WIN32
             hsDebugMessage("Error on Windows read", GetLastError());
+#else
+            hsDebugMessage("Error on POSIX read", errno);
```

Comment:
It didn't when I compiled it, but it probably doesn't hurt to include it for compliance.  Fixed